src/lib/__tests__/Preview-test.js

```diff
@@ -552,48 +580,57 @@ describe('lib/Preview', () => {
     });
 
     describe('prefetchViewers()', () => {
-        let prefetchStub;
+        describe('prefetch stubbed', () => {
+            let prefetchStub;
+
+            beforeEach(() => {
+                prefetchStub = sandbox.stub();
+                const stubViewer = () => {
+                    return { prefetch: prefetchStub };
+                };
+
+                const mockViewers = [
+                    {
+                        NAME: 'viewer1',
+                        CONSTRUCTOR: stubViewer
+                    },
+                    {
+                        NAME: 'viewer2',
+                        CONSTRUCTOR: stubViewer
+                    },
+                    {
+                        NAME: 'viewer3',
+                        CONSTRUCTOR: stubViewer
+                    }
+                ];
 
-        beforeEach(() => {
-            prefetchStub = sandbox.stub();
-            const stubViewer = () => {
-                return { prefetch: prefetchStub };
-            };
+                stubs.getViewers = sandbox.stub(preview, 'getViewers').returns(mockViewers);
+            });
 
-            const mockViewers = [
-                {
-                    NAME: 'viewer1',
-                    CONSTRUCTOR: stubViewer
-                },
-                {
-                    NAME: 'viewer2',
-                    CONSTRUCTOR: stubViewer
-                },
-                {
-                    NAME: 'viewer3',
-                    CONSTRUCTOR: stubViewer
-                }
-            ];
+            it('should prefetch no viewers if no viewer names are specified', () => {
```

Comment:
'should not prefetch viewers if none are specified'

Reply:
This was test was not modified by me, I only added the nested describe and ```should not throw when prefetching the viewers```